app/helpers/patients_helper.rb

```diff
@@ -52,4 +52,54 @@ def patient_year_group(patient, academic_year:)
   def patient_parents(patient)
     format_parents_with_relationships(patient.parent_relationships)
   end
+
+  def patient_important_notices(patient)
+    notifications = []
+
+    if patient.deceased?
+      notifications << {
+        date_time: patient.date_of_death_recorded_at,
+        message: "Record updated with child’s date of death"
+      }
+    end
+
+    if patient.invalidated?
+      notifications << {
+        date_time: patient.invalidated_at,
+        message: "Record flagged as invalid"
+      }
+    end
+
+    if patient.restricted?
+      notifications << {
+        date_time: patient.restricted_at,
+        message: "Record flagged as sensitive"
+      }
+    end
+
+    if patient.has_vaccination_records_dont_notify_parents?
+      vaccination_records =
+        patient.vaccination_records.select { it.notify_parents == false }
+
+      if vaccination_records.any?
```

Comment:
I think logically this is doing the same as `patient.has_vaccination_records_dont_notify_parents?`, do we need to do the check twice, can we remove the call to `has_vaccination_records_dont_notify_parents?`?